README

```diff
@@ -725,6 +727,11 @@ Network Support
       or
       shell$ mpirun --mca pml ucx ...
 
+  There is a known issue when using UCX with very old Mellanox Infiniband
+  HCAs, in particular HCAs preceding the introduction of the ConnectX
+  product line, which can result in Open MPI crashing in MPI_Finalize.
+  This issue will be address by UCX release 1.10 and newer.
```

Comment:
```suggestion
  This issue will be addressed by UCX release 1.10 and newer.
```

Did we find out if this was fixed in UCX 1.9 or 1.10?  There was some other PR (or issue?) where it was claimed that the issue is fixed in an upcoming 1.9 beta (and I'm not sure that 1.9 has been released yet...?).

Reply:
It is fixed in 1.9.0.  Updated that and fixed the typo.